go/mcap/testutils.go

```diff
@@ -142,7 +142,15 @@ func record(op OpCode) []byte {
 }
 
 func attachment() []byte {
-	buf := make([]byte, 9)
+	recordLen := 9 + // opcode + record length
+		8 + // record time
+		8 + // create time
+		4 + // attachment name length
+		4 + // media type length
+		4 + // data size
```

Comment:
seems like this should be 8. why doesn't this cause a parse failure? I guess because the test isn't doing anything with the CRC and is interpreting the CRC bytes as part of the data size. Since it's all zeros due to the empty content that would erroneously pass.